Week1/homework/theCatWalk/jscode.js

```diff
@@ -0,0 +1,44 @@
+
+let image;
+
+
+
+
+window.onload = () => {
+    image = document.querySelector("img");
+
+    let distance = 0;
+    let screenWidth = window.innerWidth;
+    let body = document.querySelector("body")
+
+
+
+    function catWalk() {
+        image.style.left = `${distance}px`;
+        distance += 10;
+        console.log(image);
+        console.log(distance);
```

Comment:
finalized code should not have any console.log statements. If you want to explain something in your code, try adding some comments